packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/stepfunctions/start-execution.ts

```diff
@@ -42,6 +42,15 @@ export interface StepFunctionsStartExecutionProps extends sfn.TaskStateBaseProps
    * @default - false
    */
   readonly associateWithParent?: boolean;
+
+  /**
+   * The Step Functions state machine ARN to start the execution on.
+   * This allows to specify the ARN of a specific state machine version or alias.
+   * If specified, it overrides the ARN of the stateMachine.
+   *
+   * @default - The ARN of stateMachine is used
```

Comment:
Not sure what the convention is for referencing other properties. Perhaps specify `stateMachine` is from `StepFunctionsStartExecutionProps`?

```suggestion
   * @default - The ARN of StepFunctionsStartExecutionProps.stateMachine is used
```